### PR TITLE
Description and screenshots v1

### PR DESCRIPTION
**Pull request by golovnevs**
Hello teachers! Here is the first version of description and screenshots for my final project.

### CODE REVIEW

**Review by BTBTravis — DISMISSED** [review]
Cool idea and thanks for updating the documentation! Look out for future issues where we will track the first development steps. This pull request solves #1  please merge

File: README.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+Application Name: StyleList
 
-Student Name: {student-name}
+Student Name: Golovnev Sergei
 
-## {app-name}
+## StyleList
 
-{short explanation of your application}
+My product StyleList allows users to find their basic wardrobe key pieces based on their style preferences. User goes through a quiz where she is asked to choose between 2 outfits, then the product picks 4 most frequently liked outfits and shown it to user. Then user clicks on them and the product provides list of items used on the pictures.
```
Comment (MrTim) [inline README.md]: Only 'she' can go though the quiz?)) 
Better use just user (or they https://en.wikipedia.org/wiki/Singular_they)

**Review by MrTim — DISMISSED** [review]
Good idea! Let's start :+1:

File: README.md (hunk 1 above)
Comment (golovnevs) [reply README.md]: Of course, thank you!

**Review by MrTim — APPROVED** [review]
:+1:

**Pull request merged**